embd.go

```diff
@@ -72,20 +73,36 @@ func run() error {
 		Args:  os.Args[1:],
 		Pkg:   *pkg,
 		Files: map[string]File{},
+		Dirs:  make(map[string][]File),
```

Comment:
1. Please use map[string]map[string]File, to keep it similar as for Files.
2. Please don't use make, as in Files -- change to `map[string]map[string]File{}`